tests/test_clean.py

```diff
@@ -30,7 +29,7 @@
 class TestClean(unittest.TestCase):
     def setUp(self):
         self.pwd = get_installdir()
-        self.deidpath = os.path.abspath("%s/tests/resources/" % self.pwd)
+        self.deidpath = os.path.abspath("%s/../tests/resources/" % self.pwd)
```

Comment:
This should be relative too - use the path of the file plus the path to resources instead of self.pwd.